packages/dara-components/js/graphs/graph-layout.tsx

```diff
@@ -138,6 +142,18 @@ export function parseLayoutDefinition(definition: GraphLayoutDefinition): GraphL
                 builder.nodeSeparation(definition.node_separation);
             }
 
+            if (definition.tier_separation) {
+                builder.tierSeparation(definition.tier_separation);
+            }
+
+            if (definition.tiers) {
+                builder.tiers = definition.tiers;
+            }
+
+            if (definition.orientation) {
+                builder.orientation = definition.orientation;
+            }
+
```

Comment:
Could we extract the repeated tier properties somehow? e.g. something like

```tsx
    let builder;
    switch (definition.layout_type) {
        case '{layout_type}': {
            builder = LayoutClass.Builder
            // ...
            break;
        }
        // ...
   }

   if (<builder class supports tiers>) {
     // assign tier properties
   }
```

Reply:
I have attempted this, could you double check? Planar works a bit different so I left that separate